modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/NativeFuncs.cs

```diff
@@ -42,6 +42,9 @@ private partial struct UnmanagedCallbacks
         public static partial IntPtr godotsharp_method_bind_get_method(in godot_string_name p_classname,
```

Comment:
I don't think `godotsharp_method_bind_get_method` is used anymore and could probably be removed.